# Generated for NaoQi method ALBehaviorManager.runBehavior
# Run a behavior

# The behavior directory name
string name
---


